versions/1/kibana/spec.yml

```diff
@@ -65,3 +65,25 @@
         type: file
         contentMediaType: "application/json"
         pattern: '^.+\.json$'
+    - description: Folder containing rules
+      type: folder
+      name: "rules"
+      required: false
+      contents:
+      - description: An individual rule file for the detection engine
+        type: file
+        contentMediaType: "application/json"
+        pattern: '^{PACKAGE_NAME}-rule-.+\.json$'
+      - description: Changelog for all of the rules in the folder
```

Comment:
Is it really necessary to introduce additional changelog for a package? Is it possible to use the general one for the package?

If so, could you please write a spec for this file? I suppose it's possible contrary to the rules files (as for other Kibana objects and multiple schemas).

Reply:
Lets go with a single global changelog.

Reply:
Removed this asset type, descoping the concept of a rule-wide changelog for now